File: OOP-Homework3/HW3/driver2.cpp
#include <unistd.h>
#include <iostream>
#include <cstdlib>
#include "Tetris.h"
#include "Tetromino.h"

using namespace std;



int main(){

    srand(time(0));
    
    int boardx;
    int boardy;
    
    cout << "#######     Driver2     #######\n\n";
    cout << "Terminal will be deleted in driver2 !!\n\n";
    cout << "Enter the column number of tetris board: ";
    
    while(!(cin >> boardx) || boardx <= 3){
	    cin.clear();
        cin.ignore(1000, '\n');
  	    cout << "Please enter proper input.\n";
	}
	
    cin.ignore(1000, '\n');
    
    cout << "Enter the row number of tetris board: ";
    
    while(!(cin >> boardy) || boardy <= 3){
	    cin.clear();
        cin.ignore(1000, '\n');
  	    cout << "Please enter proper input.\n";
	}
    
    cin.ignore(1000, '\n');

    MuratTetris:: Tetris board(boardy+3,boardx);

    board.play();

}



 

File: OOP-Homework 7/driver2.java
import java.util.Scanner;


public class driver2{
    public static void main(String[] args){
        int boardx=10;
        int boardy=10;
        String next;
        Scanner input1 = new Scanner(System.in);
        Scanner input2 = new Scanner(System.in);
        

        System.out.printf("#######     Driver2     #######\n\n");
        System.out.printf("Terminal will be deleted in driver2 !!\n\n");
        
        
        do{
            System.out.printf("Enter the column number of tetris board: ");
            while (!input1.hasNextInt()) {
                System.out.println("Wrong input!");
                input1.next();
            }
            boardx = input1.nextInt();
            if(boardx <= 3)
                System.out.println("Please enter a number bigger than 3!");
        }while(boardx <= 3);

        
        do{
            System.out.printf("Enter the row number of tetris board: ");
            while (!input1.hasNextInt()) {
                System.out.println("Wrong input!");
                input1.next();
            }
            boardy = input1.nextInt();
            if(boardy <= 3)
                System.out.println("Please enter a number bigger than 3!");
        }while(boardy <= 3);

        System.out.printf("%d %d", boardx, boardy);
        Tetris board = new Tetris(boardy+3,boardx);

        board.play();
    }
}